lib/runners/processor/actionlint.rb

```diff
@@ -0,0 +1,110 @@
+module Runners
+  class Processor::Actionlint < Processor
+    SCHEMA = _ = StrongJSON.new do
+      extend Schema::ConfigTypes
+
+      # @type self: SchemaClass
+      let :config, base(
+        target: target,
+
+        # Native options
+        ignore: one_or_more_strings?,
+      )
+
+      let :issue, object(
+        shellcheck_issue: object?(id: string, severity: string, line: number, column: number, message: string, url: string),
```

Comment:
[suggestion] `integer` is more suitable:

```suggestion
        shellcheck_issue: object?(id: string, severity: string, line: integer, column: integer, message: string, url: string),
```

Reply:
Changed!